test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -2873,4 +2873,23 @@ void cauldronRecorded() {
       }
     }
   }
+  @Nested
+  class Authority {
+    @Test
+    public void canDetectAssertAuthority() {
+      RequestLoggerOutput.startStream();
+      var cleanups =
+          new Cleanups(
+              withEquipped(Slot.CONTAINER, ItemPool.SHERIFF_BADGE),
+              withEquipped(Slot.CONTAINER, ItemPool.SHERIFF_MOUSTACHE),
+              withEquipped(Slot.CONTAINER, ItemPool.SHERIFF_PISTOL),
```

Comment:
these aren't required because you aren't checking for them being equipped

also they're not container slotted :P